docs/blog/2018-02-26-documentation-project/index.md

```diff
@@ -39,7 +39,7 @@ So why fix what isn’t broken?
 
 Many people have contributed to the docs and tutorials already and have done an excellent job. With that being said, there is still room to grow. 
 
-Here is a brief overview of issues that have come up again and again as I've interviewed people and observed them going through the tutorials and docs (see Issue #4175 on Github to follow this and contribute):
+Here is a brief overview of issues that have come up again and again as I've interviewed people and observed them going through the tutorials and docs (see [Issue #4175](/issues/4175/) on Github to follow this and contribute):
```

Comment:
When you link *within* a domain you can use what's called relative links e.g. if you're on example.com and you click on a link that points to `/about-me/` then your browser will go to `example.com/about-me/`.

If you want to link to *another* domain e.g. from example.com to whataboutme.com — you need to include the full domain for the *other* domain in the link e.g. `whataboutme.com/about-me/`.

So since this blog post will live on gatsbyjs.org and you want to link to github.com — you need to include the full github.com domain in the link or https://github.com/gatsbyjs/gatsby/issues/4175

Reply:
Ok thank you! I guess I thought since the blog resides in github, then they shared the same domain. So it's more about the domain where the viewer views them, not where the files are being stored?

Reply:
Yeah -- the files here are raw materials that Gatsby uses to build the site. So we need to prep materials here for what's needed on the real site. The blog isn't here. Just raw materials for the blog.